components/conforma-verifier-listener/staging/kustomization.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# Use the base configuration
+resources:
+- ../base
+
+# Staging-specific namespace
+namespace: conforma
```

Comment:
Oh I think I get it. The "default" is pre-Kustomize, so we're not ever using "default". Is that right?